File: despatch.WEB/app/src/swagger/models/contact-data-images-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { ImageResponse } from './image-response';
export interface ContactDataImagesResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  image?: ImageResponse;
  imageId?: string;
}

File: despatch.Domain/Models/Default/Response/ContactDataImagesResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataImagesResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string ImageId { get; set; }

        public virtual ContactDataResponse ContactData { get; set; }
        public virtual ImageResponse Image { get; set; }
    }
}
